tasks/install-from-source.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Clone Certbot into configured directory.
+  git:
+    repo: "{{ certbot_repo }}"
+    dest: "{{ certbot_dir }}"
+    version: "{{ certbot_version }}"
+    update: "{{ certbot_keep_updated }}"
+    force: yes
```

Comment:
I think most of @geerlingguy other install-from-source tasks either expose a `clone_depth` variable, or default it to 1 for faster clones. Maybe add it here too?

- https://github.com/geerlingguy/ansible-role-drush/blob/master/tasks/source-install.yml#L9
- https://github.com/geerlingguy/ansible-role-php-redis/blob/master/tasks/install-from-source.yml#L8
- https://github.com/geerlingguy/ansible-role-php/blob/master/tasks/install-from-source.yml#L85